using_images/s2i_images/php.adoc

```diff
@@ -27,37 +27,54 @@ either by {product-title} or by Docker.
 [[php-versions]]
 == Versions
 
-Currently, {product-title} provides version
-link:https://github.com/openshift/sti-php/tree/master/5.5[5.5] and
-link:https://github.com/sclorg/s2i-php-container/tree/master/5.6[5.6] of PHP.
+Currently, {product-title} provides versions
+link:https://github.com/openshift/sti-php/tree/master/5.5[5.5],
+link:https://github.com/sclorg/s2i-php-container/tree/master/5.6[5.6], and
+link:https://github.com/sclorg/s2i-php-container/tree/master/7.0[7.0] of PHP.
 
 [[php-images]]
 == Images
 
-This image comes in two flavors, depending on your needs:
+ifdef::openshift-online[]
+RHEL 7 images are available through Red Hat's subscription registry:
```

Comment:
s/Red Hat's subscription registry/the Red Hat Registry/

See https://access.redhat.com/containers/#/faq.